src/lib/dev-environment.js

```diff
@@ -401,15 +401,34 @@ export function getEnvironmentPath( name: string ) {
 	return path.join( mainEnvironmentPath, 'vip', 'dev-environment', name );
 }
 
-export function handleCLIException( exception: Error, slug: string ) {
+export function handleCLIException( exception: Error ) {
 	const errorPrefix = chalk.red( 'Error:' );
 	if ( 'Environment not found.' === exception.message ) {
-		const extraCommandParmas = slug ? ` --slug ${ slug }` : '';
-		const createCommand = chalk.bold( DEV_ENVIRONMENT_COMMAND + ' create' + extraCommandParmas );
+		const createCommand = chalk.bold( DEV_ENVIRONMENT_FULL_COMMAND + ' create' );
 
 		const message = `Environment doesn't exist.\n\n\nTo create a new environment run:\n\n${ createCommand }\n`;
 		console.log( errorPrefix, message );
 	} else {
 		console.log( errorPrefix, exception.message );
 	}
 }
+
+type EnvironmentNameOptions = {
+	slug: string,
+	app: string,
+	env: string,
+}
+
+export function getEnvironmentName( options: EnvironmentNameOptions ) {
+	if ( options.slug ) {
```

Comment:
Just a thought, does slug have priority if I provide both @app.env and --slug?

Reply:
It does, right? I have test case for it as well. The idea being say you would like to use an alias or have multiple dev-envs per single VIP environment.